data_adapter_oemof/mappings.py

```diff
@@ -99,7 +100,7 @@ def get_data(self, key, field_type: Optional[Type] = None):
                 )
                 return timeseries_key
             logger.warning(f"Could not find timeseries entry for mapped key '{key}'")
-            return None
+            return key
```

Comment:
If someone was to name the timeseries by hand this could be helpful and it has more meaning to it. Together with the warning I think it is easier to debug